django_etuovi/etuovi.py

```diff
@@ -34,16 +35,17 @@ def get_session() -> FTP:
     )
 
 
-def create_xml_file(items: List[BaseClass], file_path: str = ".") -> str:
+def create_xml_file(items: List[BaseClass], file_path: str = ".") -> Tuple[str, str]:
     element_tree = ElementTree(create_element_tree(items))
     filename = get_filename()
-    filename = path.join(file_path, filename)
-    element_tree.write(filename, encoding="UTF-8", xml_declaration=True)
-    return filename
+    element_tree.write(path.join(file_path, filename),
+                       encoding="UTF-8", xml_declaration=True)
+    return file_path, filename
 
 
-def send_items(filename) -> None:
+def send_items(file_path, filename) -> None:
     session = get_session()
-    session.storbinary("STOR {}.temp".format(filename), open(filename, "rb"))
-    session.rename("{}.temp".format(filename), filename)
-    session.quit()
+    with open(path.join(file_path, filename), "rb") as f:
+        session.storbinary("STOR {}.temp".format(filename), f)
+        session.rename("{}.temp".format(filename), filename)
```

Comment:
Here it might be possible to use f-strings:
```python
session.storbinary(f"STOR {filename}.temp", f)
session.rename(f"{filename}.temp", filename)
```

Reply:
it is possible, but the guide line was not to change anything if it's not broken :D